packages/account-management/src/frontend/Shared/PasswordInput.tsx

```diff
@@ -28,6 +33,9 @@ export const PasswordInput: React.FC<PasswordInputProps> = (props) => {
 
   return (
     <PasswordFieldWrapper>
+      <IconWrapper>
+        {!showPassword ? <EyeIcon onClick={toggleShowPassword} /> : <AllyVisual onClick={toggleShowPassword} />}
```

Comment:
I think I committed this line earlier today, but now I see it again, we might as well flip the logic for readability:

```suggestion
        {showPassword ? <AllyVisual onClick={toggleShowPassword} /> : <EyeIcon onClick={toggleShowPassword} />}
```

My bad 😬